src/main/kotlin/uk/gov/justice/digital/hmpps/dpssmoketest/resource/SmokeTestResource.kt

```diff
@@ -35,8 +36,7 @@ class SmokeTestResource(
   @Operation(
     summary = "Start a new smoke test for prison to probation update",
     description =
-    """
-        This tests the prison-to-probation-update happy path, which means the following scenarios are working:
```

Comment:
having a space at the beginning here makes the resulting docs switch to `code` (in purple) mode for the whole description.